### PR TITLE
Feat : Added Notecard and its attribute

### PR DESCRIPTION
**Pull request by sushil-mod**
# Added Notecard , Color tray and custom Label
how to check :- go to login -> login as a guest-> then mynotes -> then check all functionality
### jsx and Css
-  ColorPalette 
-  CreateNoteCard  
-  LabelCard  
-   NotesContext  

### screenshot
noteeditor

### CODE REVIEW

**Review by ajit-sharmaGH — APPROVED** [review]
Code looks clean
Easy to read 
Nice color combinations 
You can merge it

File: src/components/card/CreateNoteCard.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.note-title{
+    width:100%;
+    height:2.5rem;
+    border-radius:8px;
+    font-size:1.2rem;
+    outline:none;
+    border:none;
+    padding:0rem 0.5rem;
```
Comment (psychoactiv) [inline src/components/card/CreateNoteCard.css:21]: Suggestion : You can use px for values less then 10px

File: src/components/card/CreateNoteCard.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                        <div> 
+                            <input className={`note-title ${bgColor} `} type="text" placeholder="  Take notes....." />
+                        </div>
+
+                        <div className='app'>
+                            <ReactQuill bounds={'.app'}  /> 
+                        </div>
+                        <div className='note-tag-container' >
+
+                            {note.tags.map((tag,index)=><div key={index} className='note-tag' >{tag}</div> ) }
```
Comment (psychoactiv) [inline src/components/card/CreateNoteCard.jsx:33]: Suggestion: try to destructure beforehand for long chaining eg. you could've destructured tags from note beforehand.

File: src/components/card/LabelCard.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+
+    const labelHandler =(e)=>{
+        const value = e.target.value;
+        setLabel(value);
+    }
+    console.log(label);
+    const addlabel =()=>{
+                console.log("click");
```
Comment (psychoactiv) [inline src/components/card/LabelCard.jsx:17]: You can remove console.log if not needed

File: src/components/card/LabelCard.jsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                if( label !== "undefined" ){ 
+                    setLabelArray([...labelArray,label]);
+                    setLabel('');
+                }
+    }
+    console.log(labelArray);
```
Comment (psychoactiv) [inline src/components/card/LabelCard.jsx:23]: same as above

**Review by psychoactiv — APPROVED** [review]
Code was well written 
Variables were named properly 
You can merge this code

File: src/components/card/CreateNoteCard.css (hunk 1 above)
Comment (sushil-mod) [reply src/components/card/CreateNoteCard.css:21]: sure.

File: src/components/card/CreateNoteCard.jsx (hunk 2 above)
Comment (sushil-mod) [reply src/components/card/CreateNoteCard.jsx:33]: ohk , i will do that beforehand

File: src/components/card/LabelCard.jsx (hunk 3 above)
Comment (sushil-mod) [reply src/components/card/LabelCard.jsx:17]: yes

File: src/components/card/LabelCard.jsx (hunk 4 above)
Comment (sushil-mod) [reply src/components/card/LabelCard.jsx:23]: sure.

**Pull request merged**